src/ipc/ipc3/dai.c

```diff
@@ -257,7 +257,11 @@ void dai_dma_release(struct comp_dev *dev)
 	if (dd->chan) {
 		/* remove callback */
 		notifier_unregister(dev, dd->chan, NOTIFIER_ID_DMA_COPY);
+#if CONFIG_ZEPHYR_NATIVE_DRIVERS
```

Comment:
@lgirdwood  one open: not sure If we want to support ipc3 dai with native drivers?

Reply:
> @lgirdwood one open: not sure If we want to support ipc3 dai with native drivers?

Yes we need to support IPC3 as there is no current TSC decision to deprecate IPC3

Reply:
> @lgirdwood do we plan to deprecate IPC3 in the future in favor of IPC4?

That's a TSC decision, the if/when/how of IPC deprecation and planning. Good topic for next TSC though.